whitepaper/kubernetes-policy-management-whitepaper.md

```diff
@@ -18,18 +18,18 @@
+  - [Policy Management Point (PMP)](#policy-management-point-(pmp)) This represents the central management Hub that distributes policy to the fleet of desired endpoints, consolidates policy violations across fleet, integrates with enterprise tools (security operations center, incident management, GRC etc), automates remediation, provides security and compliance readiness posture, and renders contextual posture for SRE, application developer personas. 
+  - [Policy Enforcement Point (PEP)](#policy-enforcement-point-(pep))Returns results of policy evaluation in ‘inform’ mode, enforces control to desired state in ‘enforce’ mode, optionally invokes a Policy Decision Point (PDP) when the PEP is distinct from the policy evaluation engine.
+  - [Policy Reporting (PR)](#policy-reporting-(pr)) This represents how results of evaluating a policy against the state of a control are represented. Standardizing this aspect alloww results to be processed and analyzed in a consistent manner and provides ease of integration when various pieces of the policy architecture are delivered by different vendors.
 - [Policy Lifecycle](#policy-lifecycle)
-  - [Authoring](#authoring)
-  - [Versioning](#versioning)
-  - [Deployment](#deployment)
-  - [Enforcement](#enforcement)
-  - [Detection](#detection)
-  - [Remediation](#remediation)
+  - [Authoring](#authoring)Policies are authored at the PAP.
+  - [Versioning](#versioning) Versioning of policies can be easily accomplished when using GitOps approach as policies are treated just like source code and manintained in GitHub.
+  - [Deployment](#deployment)Policies are deployed by the PMP by binding them to the managed endpoints A flexible approach of doing so is by associating labels with the endpoints and applying policies to endpoint that meet certain placement rules derived from these labels.
+  - [Enforcement](#enforcement)Enforcement of policies is done by the PEP by setting the control to the desired configuration state as specified by the details within a given policy.
+  - [Detection](#detection)When a policy is specified in a non-enforce or 'inform' mode, the PEP detects any mismatches of the control state against the details speified in the policy and conveys these results to the PMP.
+  - [Remediation](#remediation)Remediation involves fixing any policy violations so the control in question is operating to best practices specified in the policy. When the policy is specified in 'enforce' mode, the PEP perfosms remediation when it finds any mismatches. When the policy is specified in 'inform' mode, the PEP reports mismatches as policy violations to the PMP which can then trigger remediation or route an alert to an incident management system or security operations center (for security related policies) which in turn can initiate remediation acions.
```

Comment:
small typo - "perfosms" vs performs.  Also would not define specific implementation details here.

Reply:
fixed typo